src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java

```diff
@@ -480,6 +482,81 @@ public void ascii() throws IOException {
         );
     }
 
+    @Test
+    public void ifTest() throws IOException {
+        assertThat(
+                executeQuery("SELECT IF(age > 30, 'True', 'False') AS Ages FROM " + TEST_INDEX_ACCOUNT
+                        + " WHERE age IS NOT NULL GROUP BY Ages", "jdbc"),
+                containsString("\"type\": \"keyword\"")
+        );
+        assertThat(
+                executeQuery("SELECT IF(2 > 0, 'hello', 'world') AS if FROM " + TEST_INDEX_ACCOUNT),
+                hitAny(kvString("/fields/if/0", equalTo("hello")))
+        );
+        assertThat(
+                getHits(
+                executeQuery("SELECT IF(2 = 0, 'hello') AS if FROM " + TEST_INDEX_ACCOUNT)
+                ).getJSONObject(0).query("/fields/if/0"),
+                equalTo(null)
+        );
+    }
+
+    @Test
+    public void ifnull() throws IOException {
+        assertThat(
+                executeQuery("SELECT IFNULL(lastname, 'unknown') AS name FROM " + TEST_INDEX_ACCOUNT
+                        + " GROUP BY name", "jdbc"),
+                containsString("\"type\": \"keyword\"")
+        );
+        assertThat(
+                executeQuery("SELECT IFNULL('sample', 'IsNull') AS ifnull FROM " + TEST_INDEX_ACCOUNT),
+                hitAny(kvString("/fields/name/0", equalTo("sample")))
+        );
+
+        //TODO: let null pass the verification of semantic analyzer
```

Comment:
either remove commented code or add it as @Ignore test with reference to github issue that will allow to fix that problem